src/index.html

```diff
@@ -15,8 +19,110 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin=" "
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="conteiner">
+      <a
+        class="img"
+        href="https://s3-alpha-sig.figma.com/img/833f/9aab/
+          47d660efd384318772b6b8040db831aa?Expires=1712534400&Key-
+          Pair-Id=APKAQ4GOSFWCVNEHN3O4&Signature=oTn3~L--
+          mDnRZKcoPxL8QydsYHXYco4Bp8Rgm3qnrxJZul2VRiIySVkE-
+          22IvcmcudEcgEsHNLsumGF5NzOa1lMBtAM93RiLNbEJeDlxn
+          MRmF03MOXKmzBMbjOY2MeO13BtDRlmFkfsa8sgGcfZY~znCgT0k
+          PtoikEXAlTqXBW2rfUXrR9wmSdYCxkfqbSNt6YOd1KGvgcA0HpAev
+          ZkzZCtJojMr3tYZBgJtAaAlWNbmBrkwdOlEEacUBFdQLY1F7DT31l6ja8mwCaZEf
+          GgiwDsoHFRQmzDY6I1XCZYJHfY93kZH08u-J~PXvuTSG4rb7Q2nZsle7IDz2QYt7tXYrQ__"
+      >
+        <img
+          class="img1"
+          href="moyo_1.png"
+          src="./images/logo.png"
```

Comment:
Consider pushing and deploying the image 
<img width="1694" alt="Screenshot 2024-05-24 at 16 27 06" src="https://github.com/mate-academy/layout_moyo-header/assets/135232001/7a235d6f-0728-4acd-a009-a22de3e8180c">